src/get-started/setup-an-ide.md

```diff
@@ -1,5 +1,7 @@
 # Setup an IDE
 
+Using an IDE is recommended as it makes it a lot easier to for example see the fields of a struct. However it is not required to use a specic IDE or editor.
```

Comment:
what is "it is not required to use a specic IDE or editor"? even change "specic" to "specific" it still confuses me.

Reply:
```suggestion
Using an IDE is recommended as it makes it a lot easier to work with the TiDB code, for example to see the fields of a struct. However it is not required to use a specific IDE or editor. You can use the IDE or editor of your choice.
```

Reply:
Thanks for your update. I get the point.